webrtc/common_audio/BUILD.gn

```diff
@@ -120,7 +120,7 @@ rtc_source_set("common_audio_c_arm_asm") {
   }
 }
 
-rtc_source_set("common_audio_c") {
+rtc_static_library("common_audio_c") {
```

Comment:
This is okay for m62 but we will not want to convert m71 into static libraries because m71 merges objs together automatically.

Reply:
Sure, those changes won't be merged to m71. I will keep that in mind :)